### DISCUSSION TITLE
How can I run the Errsole Web Dashboard as a standalone app, separate from my main app?

### DISCUSSION THREAD

**Discussion by venki91** [errsole/errsole.js#113] [FAQs]
### 1. Disable the Web Dashboard in your main app

In your main app's Errsole configuration, set `enableDashboard` to `false`. This ensures that Errsole only collects and stores logs without starting the web dashboard.

```javascript
errsole.initialize({
  storage: new ErrsoleSQLite('path/to/logs/file.sqlite'),
  appName: 'your-app-name',
  enableDashboard: false
});
```

### 2. Run the Web Dashboard as a standalone app

On a separate server or the same server, create a new app and configure Errsole with `enableDashboard` set to `true`. Use the same storage configuration as your main app so this standalone app can access the logs generated by your main app. Once set up, you can access the web dashboard from this new app, independent of your main app.

#### dashboard.js
```javascript
const errsole = require('errsole');
const ErrsoleSQLite = require('errsole-sqlite');

errsole.initialize({
  storage: new ErrsoleSQLite('path/to/logs/file.sqlite'),
  appName: 'your-app-name',
  enableDashboard: true
});
```

```bash
node dashboard.js
```

For more details, refer to the [Setup](https://github.com/errsole/errsole.js/blob/master/README.md#setup) documentation.

**Comment by dejf-js** [comment]
Thanks. Got it. The ersole-web is just `index.mjs` with:
```javascript
import errsole from "errsole";
import ErrsolePostgres from "errsole-postgres";

errsole.initialize({
  storage: new ErrsolePostgres({
    host: "localhost", // Replace with your actual PostgreSQL host
    user: "postgres", // Replace with your actual PostgreSQL user
    password: "postgres", // Replace with your actual PostgreSQL password
    database: "errsole", // Replace with the name of your PostgreSQL database
  }),
});
```
